src/content/docs/style-guide/word-choice/usage-dictionary.mdx

```diff
@@ -272,9 +272,9 @@ For consistency across New Relic content, here are our general word usage guidel
     id="call-called"
     title="calls, calling, and called by"
   >
-    When describing how one entity communicates or is connected to another, focus on the calls and which direction they’re going. For example, “service A calls service B,” “called services,” “calling services.”
+    When describing how one entity communicates or is connected to another, focus on the calls and which direction they’re going. For example, `service A calls service B`,` `called services`, `calling services`.
```

Comment:
This sentence loses me, and the `s might be causing the build issues . What are you trying to convey with the back half of the `For example`?